src/wp-includes/blocks.php

```diff
@@ -949,12 +949,12 @@ function insert_hooked_blocks( &$parsed_anchor_block, $relative_position, $hooke
  * @since 6.5.0
  * @access private
  *
- * @param array                   $parsed_anchor_block The anchor block, in parsed block array format.
- * @param string                  $relative_position   The relative position of the hooked blocks.
- *                                                     Can be one of 'before', 'after', 'first_child', or 'last_child'.
- * @param array                   $hooked_blocks       An array of hooked block types, grouped by anchor block and relative position.
- * @param WP_Block_Template|array $context             The block template, template part, or pattern that the anchor block belongs to.
- * @return string An empty string.
```

Comment:
Hmm, shouldn't we keep this?

Reply:
Hmm, I don't actually remember removing this 😕 Let me add it back in.

Reply:
https://github.com/WordPress/wordpress-develop/pull/6358/commits/0f59329d6529b168598c6554a16f917314a103a1